modelci/types/vo/model_vo.py

```diff
@@ -69,6 +72,13 @@ class Status(CaseInsensitiveEnum):
     FAIL = 'Fail'
 
 
+class ModelStatus(CaseInsensitiveEnum):
+    PUBLISHED = 'Published'
```

Comment:
**1. I dont like the design. It is very confusing**

published means what? all models in our system can be regarded as the published

`training` means in training or `under revision`

`ready` for what?

`in service` means the service is running or just a model has been converted and bound to a serving system

**2. please add some comments**

Reply:
PUBLISHED means the modified model is registered and wait to be trained
TRAINING means in training 
READY status means models are ready to be deployed,  newly registered pre-trained models and those finished training are marked as READY(including converted models)
IN SERVICE means models are deployed

Reply:
not clear.

think of the whole workflow of modelci 

publish - convert - profiling ... edit ...

each status should be matched to one or multiple stages in chronologically order.

Reply:
How about using:
`draft`: edit, not trained yet
`training`: ML training
`published`: training finished, the status includes all publish - convert - profiling - deploy stages. Use the single status is because when one copy of the published model is converting, its another copy can under profiling at the same time, and another copy can be deployed.

Reply:
for tracking workflow stage in `publish - convert - profiling - deploy`, it should use workflow manager page

Reply:
@YuanmingLeee please think of this deeply and organize a better definition like this 

![image](https://user-images.githubusercontent.com/5894780/106995856-c31d7e00-67ba-11eb-879f-28eb2cc42fd1.png)

Reply:
### Workflow Status

The ModelCI workflow status values are described below:

`published`: model published to hub and ready for conversion, profiling, and deployment

`converting`: model is under conversion

`profiling`: model is under profiling

`deployed`: model is deployed as a service

`draft`: model is under edit

`validating`: model is under quick accuracy validation

`training`: model is under training

The training phase status contains: `draft` - `validating` - `training` - `published`

The serving phase status contains: `published` - `converting` - `profiling` - `deployed`